pkgs/by-name/ya/yaak/package.nix

```diff
@@ -95,13 +106,25 @@ rustPlatform.buildRustPackage (finalAttrs: {
       ln -s ${protobuf}/include src-tauri/vendored/protoc/include
     '';
 
+  tauriBuildFlags = [
+    "--config"
+    "./src-tauri/tauri.commercial.conf.json"
+  ];
+
   # Permission denied (os error 13)
   # write to src-tauri/vendored/protoc/include
   doCheck = false;
 
   preInstall = "pushd src-tauri";
 
-  postInstall = "popd";
+  postInstall =
```

Comment:
Borrowed from #424876, @dibenzepin if  you want me to remove it or cherry-pick your commit I can do that